core/utils/codel.h

```diff
@@ -148,6 +154,35 @@ class Codel final: public Queue<T> {
 
   size_t Size() override { return queue_.size(); }
 
+  int Resize(size_t new_capacity) override {
+    if (new_capacity <= Size()) {
+      return -1;
+    } else if (new_capacity >= queue_.max_size()) {
+      return -1;
```

Comment:
Just document it -- it looks strange to have a resize method which mostly does nothing.